eclipse/sw360
# feat(ProjectUI):Add Clearing Request button in project Details page

Signed-off-by: Abdul Kapti <abdul.mannankapti@siemens.com>

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
 * Enable `Create Clearing Request` button under `Clearing Status` tab of project details view.

Issue: #875 

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
* User should be able to `create/view` the `Clearing Request` from `Clearing Status` tab in project details page. (as shown in below screenshot)
* Also, test that existing functionality of creating the `Clearing Request` from `project list page` is not impacted

![viewCR](https://user-images.githubusercontent.com/50870174/84226701-2659f200-ab00-11ea-9d09-7d98c3cd312c.png)
![createCR](https://user-images.githubusercontent.com/50870174/84226705-2954e280-ab00-11ea-84f1-d6bcd93a8efe.png)


### Checklist
Must:
